### ISSUE TITLE
link editions with Wikisource through sitelinks

### ISSUE THREAD

**Issue by bodhisattwawiki** [inventaire/inventaire#774]
Wikisource contains editions of literary works and are linked with Wikidata through sitelinks. It would be nice to link editions to display Wikisource logo and link back to them, where they are applicable. 

For example, [this edition](https://inventaire.io/entity/wd:Q110818679)  on inventaire can display a Wikisource logo of the [text](https://bn.wikisource.org/s/i12z) through [this sitelink](https://www.wikidata.org/wiki/Q110818679)

**Comment by maxlath** [comment]
Yes! It was hard to until recently, as we had the Wikidata edition quarantine #182, but now that should totally be doable

**Comment by jum-s** [comment]
Wikisource links are now displayed, thanks to commit 9f31ad8d5
Thx @belett for the bump !
